examples/vpc-endpoints/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "region" {
+  type = string
```

Comment:
please add `description` to all variables and all check description for all outputs